doc/user_manual/postprocessor.tex

```diff
@@ -149,6 +149,19 @@ \subsubsection{BasicStatistics}
   quantities are going to be calculated, if \textit{False} unbiased.
   \default{False}
   %
+  \item \xmlNode{dataset}, \xmlDesc{string (boolean), optional field}, if \textit{True} \xmlString{DataSet}
```

Comment:
is boolean.
The user should not care how we read it :D

Reply:
changed